lib/LibManifestPlugin.js

```diff
@@ -30,19 +30,24 @@ class LibManifestPlugin {
 				const manifest = {
 					name,
 					type: this.options.type,
-					content: chunk.modules.reduce((obj, module) => {
+					content: chunk.mapModules(module => {
 						if(module.libIdent) {
 							const ident = module.libIdent({
 								context: this.options.context || compiler.options.context
 							});
 							if(ident) {
-								obj[ident] = {
-									id: module.id,
-									meta: module.meta,
-									exports: Array.isArray(module.providedExports) ? module.providedExports : undefined
+								return {
+									ident,
+									data: {
+										id: module.id,
+										meta: module.meta,
+										exports: Array.isArray(module.providedExports) ? module.providedExports : undefined
```

Comment:
True, this will be a future refactoring to Set. Currently it can be `Array` `null` or `true` so a truthy check is not possible currently.